src/test/java/uk/gov/hmcts/reform/jps/controllers/SittingRecordControllerTest.java

```diff
@@ -145,70 +153,25 @@ void shouldReturnResponseWithSittingRecordsWhenNoRecordsForGivenCriteria() throw
-            .changeByUserId("11244")
-            .changeDateTime(LocalDateTime.now().minusDays(2))
-            .changeByName("Matt Murdock")
-            .build();
-        StatusHistory statusHistory2b = StatusHistory.builder()
-            .statusId(StatusId.PUBLISHED.name())
-            .changeByUserId("11245")
-            .changeDateTime(LocalDateTime.now().minusDays(1))
-            .changeByName("Peter Parker")
-            .build();
-        StatusHistory statusHistory2c = StatusHistory.builder()
-            .statusId(StatusId.SUBMITTED.name())
-            .changeByUserId("11246")
-            .changeDateTime(LocalDateTime.now())
-            .changeByName("Stephen Strange")
-            .build();
-        SittingRecord sittingRecord2 = SittingRecord.builder()
-            .sittingRecordId(2L)
-            .statusId(statusHistory2c.getStatusId())
-            .createdDateTime(statusHistory2a.getChangeDateTime())
-            .createdByUserId(statusHistory2a.getChangeByUserId())
-            .createdByUserName(statusHistory2a.getChangeByName())
-            .changeDateTime(statusHistory2c.getChangeDateTime())
-            .changeByUserId(statusHistory2c.getChangeByUserId())
-            .changeByUserName(statusHistory2c.getChangeByName())
-            .build();
-        sittingRecord2.setStatusHistories(List.of(statusHistory2a, statusHistory2b, statusHistory2c));
-
-        List<SittingRecord> sittingRecords = List.of(sittingRecord1, sittingRecord2);
+        List<SittingRecord> sittingRecords = generateSittingRecords();
+        List<RecordingUser> recordingUsers = generateRecordingUsers();
 
         when(sittingRecordService.getTotalRecordCount(isA(SittingRecordSearchRequest.class),eq(SSCS)))
             .thenReturn(sittingRecords.size());
         when(sittingRecordService.getSittingRecords(isA(SittingRecordSearchRequest.class), eq(SSCS)))
             .thenReturn(sittingRecords);
+        when(statusHistoryService.findRecordingUsers(anyString(), anyString(), anyList(), any(), any()))
+            .thenReturn(recordingUsers);
 
         String requestJson = Resources.toString(getResource("searchSittingRecords.json"), UTF_8);
         MvcResult mvcResult = mockMvc.perform(MockMvcRequestBuilders.post(
-                                                      "/sitting-records/searchSittingRecords/{hmctsServiceCode}",
+                                                 "/sitting-records/searchSittingRecords/{hmctsServiceCode}",
                                                       SSCS
                                                   )
                                                   .contentType(MediaType.APPLICATION_JSON)
                                                   .content(requestJson)
-            ).andDo(print())
+            )
+            .andDo(print())
             .andExpect(status().isOk())
             .andExpect(content().contentType(MediaType.APPLICATION_JSON))
             .andReturn();
```

Comment:
assertion required to check whether RecordingUsers is set in the response.

Reply:
Done